### ISSUE TITLE
IEEE1516e project has many errors/warning upon gradle install

### ISSUE THREAD

**Issue by PHameete** [MSG134/IVCT_Framework#105]
IEEE1516e project has many errors/warning upon `gradle install`

Is this project still used, or will it be needed again when removing the hard-coded pRTI dependency?

**Comment by rhzg** [comment]
It is needed for compiling on machines without any RTI installation. 

The compiler warning should be prevented by either using the right compiler options or adding @SuppressWarnings in the code. The second option has the disadvantage to change the original IEEE code. I would vote vor option 1

**Comment by rhzg** [comment]
is solution only working on IntelliJ?

**Comment by PHameete** [comment]
No, should work for eclipse as well. The fix is in the gradle config.
